src/annotator/util/test/extract-annotation-query-test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var annotationIds = require('../extract-annotation-query');
+
+describe('annotation queries', function () {
+  var annotation = annotationIds.extractAnnotationQuery('http://localhost:3000#annotations:alphanum3ric_-only');
```

Comment:
We have a utility called `unroll` for writing data-driven tests which I think would be useful here. See [this test](https://github.com/hypothesis/client/blob/master/src/sidebar/test/tabs-test.js#L29) for an example.